### DISCUSSION TITLE
Vector Lines Have Various Bugs

### DISCUSSION THREAD

**Discussion by dootdoote** [keiwando/toonsquid#124] [General]
Hi! Just wanted to say, I absolutely adore this app and all the features it provides. I've shared it around with all my animator friends and they all love it.

The only major issue I've noticed is the current instability of Vector Lines, specifically when applied to the dynamic brush. I've observed that some strokes cannot be edited after a certain period of time and some strokes cannot be filled in even when they are closed. These issues are minor on the surface but make animation cleanup difficult, especially when coloring. I can provide examples and videos if necessary.

Again, love the app and its potential. Thank you so much for what you're providing to the animation community!

**Comment by ArtistBirdy** [comment]
@keiwando 
Stroked of vector paths are shaky even when i animate simple shapes
 
![image](https://user-images.githubusercontent.com/118920457/210539435-56db0b43-0390-4627-b0d3-818310890c97.gif)

**Reply by dootdoote** [reply]
it seems like vectors in general are a tad bit unstable, which really is my only problem with the app thus far. been a huge fan of everything else

**Reply by keiwando** [reply]
@ArtistBirdy Please send the project file with those shaky lines to contact@toonsquid.com and I'll gladly take a look at what might be happening there.

**Reply by h-r-u-m** [reply]
I have found a workaround that helped me in the same situation, try to change the wiggling path points  to "Disconnected"

**Comment by keiwando** [comment]
Thank you very much for sharing the app, I appreciate it a lot!

Can you maybe clarify what kind of "edits" you mean by 
> I've observed that some strokes cannot be edited after a certain period of time

> and some strokes cannot be filled in even when they are closed.
Could you please send me the project file in which this happens? That would be very helpful! Unfortunately, just a screen capture will not be enough for me to be able to tell what exactly is causing that calculation to fail.
